streampipes-data-explorer/src/main/java/org/apache/streampipes/dataexplorer/DataExplorerSchemaManagement.java

```diff
@@ -126,34 +117,20 @@ public void deleteMeasurement(String elementId) {
   @Override
   public boolean deleteMeasurementByName(String measureName) {
     boolean isSuccess = false;
-    CouchDbClient couchDbClient = Utils.getCouchDbDataLakeClient();
-    List<JsonObject> docs = couchDbClient.view("_all_docs")
-                                         .includeDocs(true)
-                                         .query(JsonObject.class);
-
-    for (JsonObject document : docs) {
-      if (document.get("measureName")
-                  .toString()
+    List<DataLakeMeasure> docs = dataLakeStorage.getAllDataLakeMeasures();
+
+    for (var measure : docs) {
+      if (measure.getMeasureName()
                   .replace("\"", "")
```

Comment:
I'd assume that I can replace the two `replace()` statements as well. What do you think? Do I miss anything?

Reply:
Sorry, I am also not quite sure why they are there.

Reply:
Performed some investigation: The replace statements are not necessary (anymore).